apps/gauzy/src/app/pages/integrations/components/plugin-builtin/plugin-builtin.component.scss

```
@@ -0,0 +1,213 @@
+    }
+  }
+
+  &.coming-soon {
+    opacity: 0.6;
+    cursor: default;
+  }
+}
+
+.card-logo {
+  width: 100%;
+  padding-top: 75%; // 4:3 aspect ratio
```

Stylelint (17.4.0)

[error] Unknown word 4 (CssSyntaxError)

(CssSyntaxError)